CMSgov/design-system
# [WNMGDS-570] Update installation, css, font image documentation

### Summary
- Updates "Install with NPM" page to be "Installation"
- Merge "Using font and images" with the "Using Sass/CSS" page
- Add additional font and image path examples 
- Move browser support page to guidelines

### How to test
- Run the doc site and ensure all the content and links are working
- Review the content
